sdk/core/azure-core/src/test/java/com/azure/core/implementation/InterceptorManager.java

```diff
@@ -1,13 +1,13 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-package com.azure.core.test;
+package com.azure.core.implementation;
```

Comment:
Do we want this under implementation? Doesn't this Force us to export this in the module-info again to every other module that will use it?

Reply:
Same goes for other classes accessed outside of Azure Core. I could see a refactoring to have TestBase have wrapper method around this and other classes such as TestResourceNamer.

Reply:
I'm not as worried about this given these are test classes and not part of the public API or module system.

Reply:
Could we move them to `implementation.test`?

Reply:
Since these are in `src/test` directory, `exports <packagename>` will not include these classes. So, `implementation` or `implementation.test` doesn't make a difference.